RecoLocalCalo/HcalRecAlgos/src/HFSimpleTimeCheck.cc

```diff
@@ -0,0 +1,133 @@
+#include <cstring>
+#include <climits>
+
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFSimpleTimeCheck.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFRecHitAuxSetter.h"
+
+HFSimpleTimeCheck::HFSimpleTimeCheck(const std::pair<float,float> tlimits[2],
+                                     const float energyWeights[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
```

Comment:
I believe you can pass this by reference as 
const float (&energyWeights)[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],